### ISSUE TITLE
Cannot use `graphics-document` role with `aria-roledescription`.

### ISSUE THREAD

**Issue by Kretajak** [dequelabs/axe-core#3169]
Axe is reporting `Use aria-roledescription on elements with a semantic role` when using role `graphics-document` along with  `aria-roledescription`.

Product: axe-core 4.3.3

Expectation: Issue should not be raised 

Actual: Axe is reporting `Use aria-roledescription on elements with a semantic role` with comment: `Ensure aria-roledescription is only used on elements with an implicit or explicit role`.

Motivation: Element has defined semantic role so issue should not show up.

Steps to reproduce:

Validate with axe: 
```html
      <div role="graphics-document" aria-roledescription="visualization">
        <canvas width="200" height="200"></canvas>
      </div>
```

Or enter https://ecqc6.csb.app/ and use Chrome axe devtools extension and run check.

**Comment by straker** [comment]
Thanks for the issue. I'll bring it up with @WilcoFiers but here are the results of my testing:

```html
<div style="border: 1px solid;" role="graphics-document" aria-roledescription="visualization" tabindex="0">
    <canvas width="200" height="200"></canvas>
  </div>


  <div style="border: 1px solid red;" role="graphics-document" tabindex="0">
    <canvas width="200" height="200"></canvas>
  </div>
```

- Safari / VO - reads the `graphics-role` and respects the `aria-roledescription`
- IE11 / JAWS - does not read the role or the roledescription
- Chrome / JAWS - does not read the role or the roledescription
- Firefox / NVDA - does not read the role or the roledescription

**Comment by straker** [comment]
Due to lack of support in most browsers, this is working as intended so going to close. If you need to get it to work you can configure axe to [update the list of supported roles](https://github.com/dequelabs/axe-core/blob/develop/lib/checks/aria/aria-roledescription.json#L5-L13) for the check to include `graphics-document`.
